### ISSUE TITLE
Startup timeout settings are confusing

### ISSUE THREAD

**Issue by mmc2004jp** [cloudfoundry/docs-dev-guide#36]
In large-app-deploy.html page, there is explanation about app startup timeout settings.
I think it's currently hard to understand.

> CF_STARTUP_TIMEOUT: Controls the maximum time that the cf CLI waits for an app to start. Value set in minutes.
> cf push -t TIMEOUT: Controls the maximum time that the cf CLI waits for an app to start. When you use this flag, cf ignores any app start timeout value set in the manifest or in the CF_STARTUP_TIMEOUT environment variable. Value set in seconds.

According to the description, it looks these 2 timeout settings (actually 3 settings including timeout attribute in manifeset.yml) have completely same meaning, because they have the same explanation ("Controls the maximum time that the cf CLI waits for an app to start"). However, they have different default value (5 min vs 60 sec). Therefore I think this is confusing for readers (Why do different settings exist for one purpose? There are 2 default values, so if I specify nothing, which value is used?).

I was considering what these 3 timeout settings actually mean in following thread:
https://groups.google.com/a/cloudfoundry.org/forum/#!topic/vcap-dev/R4ZtcWo6B0g

My current understanding is:
- CF_STARTUP_TIMEOUT controls the maximum time that the cf CLI waits for an app to start.
- cf push -t TIMEOUT (and timeout attribute in manifest.yml) controls the maximum time that the cf CLI waits for an app to start. This also controls maximum time that Cloud Foundry waits for an app to start at server side.

In short, CF_STARTUP_TIMEOUT is for client only, but cf push -t TIMEOUT and timeout in manifest are for both client and server (I think).

If my understanding is correct, I think the article will be much easy to understand by adding the information (difference of them). For example, I would suggest to add explanation like below:

> cf push -t TIMEOUT: Controls the maximum time that the cf CLI waits for an app to start.

to

> cf push -t TIMEOUT: Controls the maximum time that the cf CLI waits for an app to start. This also controls the maximum time that Cloud Foundry waits for an app to start at server.

**Comment by animatedmax** [comment]
29 Sept 2015 - Updated documentation - https://www.pivotaltracker.com/story/show/91089386

https://github.com/cloudfoundry/docs-dev-guide/commit/17b4604ba4e3268d0cee2c48161292169aa7819d

https://github.com/cloudfoundry/docs-dev-guide/commit/6d10d019dbae3cee2e1e83b141246c1e3102b109
